api/app/GraphQL/Mutations/UpdateUserSkillRankings.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace App\GraphQL\Mutations;
+
+use App\Models\User;
+
+final class UpdateUserSkillRankings
+{
+    /**
+     * Update a user's UserSkill rankings
+     *
+     * @param  array{userId: UUID, userSkillRanking: UpdateUserSkillRankingsInput}  $args
+     * @return User
+     */
+    public function __invoke($_, array $args)
+    {
+        $user = User::find($args['userId'])->load(['userSkills', 'userSkills.skill']);
+        $userSkillsCollection = $user->userSkills;
```

Comment:
You don't need to load anything yet, since you don't use userSkillsCollection until after you've refreshed user anyway